test/e2e/suite_test.go

```diff
@@ -284,7 +288,7 @@ func e2eTest(ctx context.Context, opts *e2eTestOpts) func(*testing.T) {
 		setTestNamespaceNames(opts)
 
 		err = beforeTest(t, f, opts)
-		defer afterTest(t, f, opts)
+		//defer afterTest(t, f, opts)
```

Comment:
Did that line create an issue or did you forget to uncomment?

Reply:
Good catch. I forgot. I need to stop being lazy and add an option to skip cleanup. I'll create a ticket for that.